simpledotorg/simple-server
# Show Follow ups v2 numbers via a feature flag

**Story card:** [ch6160](https://app.shortcut.com/simpledotorg/story/6160/spike-out-follow-ups-from-matviews-to-support-v2) and [ch3818](https://app.shortcut.com/simpledotorg/story/3818/change-follow-up-details-table-in-region-reports-to-track-visits)

## Because

We want to change follow ups to report on visits instead of just BPs taken.

## This addresses

* Changed the follow ups matview to report on any visit, and not just BPs taken
* Grabbing numbers from the follow ups matview if the `follow_ups_v2` feature flag is enabled
* tests / FF changes

## How to test

First, create the feature `follow_ups_v2` in the Flipper UI
Then, turn on the flag via a query string for any reporting page where you want to see the new numbers.  For example, in my dev server I do:

http://localhost:3000/reports/regions/facility/hwc-clove-village/details?_follow_ups_v2=1
